### DISCUSSION TITLE
Color of Input-Component

### DISCUSSION THREAD

**Discussion by a-smily** [tasteui/tasteui#445] [Q&A] [answered]
Hi, how can i change the bg-color or text-color of the input-component?
I can't get it work with Personalization Concepts.. can you give a hint?

**Comment by devajmeireles** [comment, marked as answer]
Have you tried using soft personalization with the `append` or `prepend` strategy? https://tallstackui.com/docs/personalization/soft#interacting-with-default-classes

I think it will work, you need to point this to the appropriate block. Let me know if you have any questions.

**Reply by a-smily** [reply]
I tried it with the "Scoped Personalization". 
The problem I have is where to find the tailwind class-elements of the component which are in use so i can replace or append them or point this to the appropriate block.
I want to adjust the colors of the components for a theme.. maybe there is another approach?
Thanks for this library and your help :)

**Comment by Anatolii-Ovcharuk** [comment]
You also can look script in repository: https://github.com/Anatolii-Ovcharuk/JS-Module-Code
And look how it working here: https://anatolii-ovcharuk.github.io/JS-Module-Code/
